tests/containers/validate_btrfs.pm

```diff
@@ -41,7 +44,18 @@ sub _sanity_test_btrfs {
     }
     assert_script_run("echo -e 'FROM $img\\nENV WORLD_VAR Arda' > $dockerfile_path/Dockerfile");
     my $btrfs_head = '/tmp/subvolumes_saved';
-    $rt->info(property => 'Driver', value => 'btrfs');
+
+    if (version->parse(get_docker_version()) >= version->parse('23.0.5')) {
+        $rt->info(property => 'Driver', value => 'overlay2');
+        systemctl('restart docker');
+        script_output('cat /etc/docker/daemon.json');
+        assert_script_run(q[sed -i 's/{/{ "storage-driver": "btrfs",/' /etc/docker/daemon.json]);
```

Comment:
smart :)
But what if the default daemon.json is empty? (not sure if this could ever happen)